### DISCUSSION TITLE
Post-Digital Therapy to Repair the Sense of Belonging [Mayue]

### DISCUSSION THREAD

**Discussion by myyyyy481** [realitydeslab/playshop-summer-2024#25] [Proposal]
Chinese Title:用后数字疗法来修复归属感

research problem:
In the context of globalization, migration and cultural diversity, how are people in a foreign land finding belonging? How do communicative people adapt to a diverse social environment? How do we use Post-Digital Therapy to help them deal with negative emotions and build a sense of belonging?
在全球化、移民和文化多样性的背景下，身处异乡的人如何找到归属感？不善沟通的人如何适应多样化的社会环境？我们如何利用后数字疗法帮助他们处理负面情绪，建立归属感？

Key words:
Identity,  Social interaction,  Globalization,   Emotional Value
身份认同、社会互动、全球化、情感价值；

Artefact / Media:
image，video：Ai generation
图片，视频：ai生成

Abstract:
   Floating society has brought economic prosperity and cultural exchange, but also brought the lack of sense of belonging of people in different places.
The project combines natural linkage algorithms, ELLM technology and AI to provide people with real-time emotional detection and emotional help services lacking a sense of belonging. The database collects users 'emotional and memory data, AI senses users' psychology in real time, and timely feedback the lack of sense of belonging to the system. The system analyzes the scene images required by users, generates dialog boxes and image symbols in real time through MR technology to enable users to get rid of negative emotions, and gives suggestions to users, encourages users to communicate with others, and establishes a sense of belonging.
This project aims at enable technology to better understand and adapt to the communication methods of groups with different cultural backgrounds, promote the boundaries of human-computer interaction through innovative technologies, improve the satisfaction of users when interacting with AI, and help people without a sense of belonging to rebuild their sense of belonging in different places.
流动社会带来了经济繁荣和文化交流，也带来了不同地方人们归属感的缺失。
该项目结合了自然链接算法、ELLM技术和人工智能，为人们提供实时的情感检测和缺乏归属感的情感帮助服务。该数据库收集用户的情感和记忆数据，人工智能实时感知用户的心理，并及时反馈用户对归属感的缺失。系统分析用户所需的场景图像，通过Mr技术实时生成对话框和图像符号，使用户摆脱负面情绪，向用户提供建议，鼓励用户与他人沟通，建立归属感。
这个项目旨在使技术更好地理解和适应群体与不同文化背景的沟通方式，通过创新技术促进人机互动的边界，提高用户与人工智能互动的满意度的同时，帮助缺失归属感的人群重建他们的归属感。

Visual Reference / Illustrations：

![videoframe_1488](https://github.com/realitydeslab/playshop-summer-2024/assets/173876622/4c5d0972-1642-4c10-acfa-32b5cac13daf)
![keyframe  马越](https://github.com/realitydeslab/playshop-summer-2024/assets/173876622/88aa3570-1e00-41f1-bac5-adf0f0cf3f6d)
![videoframe_755](https://github.com/realitydeslab/playshop-summer-2024/assets/173876622/f69f5843-2733-4480-b589-e77c72bc7b75)

![videoframe_3448](https://github.com/realitydeslab/playshop-summer-2024/assets/173876622/beccaa86-93e6-4ab8-ad52-e223846a8e3b)

slogan：
"Digital Harbor"-The lack of sense of belonging of people from different places in the mobile society and how to rebuild their sense of belonging and help them integrate into the new environment.
“数字港湾”——移动社会中来自不同地方的人缺乏归属感，如何重建他们的归属感并帮助他们融入新环境。

prompt for a 15 seconds video clip:

At the party of a foreign university, an international student girl felt out of place with others because of introversion and cultural differences. After wearing vr glasses of "digital sense of belonging" system, she saw the cultural elements of her hometown at the party. Through the encouragement and guidance of ai dialog box, she integrated into the party, made new friends and established a sense of belonging.

key frame：

[video-kjsj.zip](https://github.com/user-attachments/files/16094334/video-kjsj.zip)

References:
Bibliography:
(1).Benedict Anderson's《 Imaginary Community》
(2).Yuval-Davis's《The Politics of Belonging》
(3).Alan Cowen-”Semantic Space Theory”《Trends in Cognitive Sciences》
(4).Alan Cowen-Empathic Large Language Model (eLLM)
(5).(Hope, tolerance and empathy: employees’ emotions when using an Al-enabled chatbot in a digitalised workplace)   
     Information Technology & PeopleSCI 2.2022.
(6).LYUXingyang.YANG Yuti.XU Shmangyu(With intelligence: a study on the remedial effect of artiffcial intelligence)
(7).XU linlan.WENGchunyin("Why to be a home": the youth in a mobile society Local studies of digital vagrants)

**Comment by glitch-kraft** [comment]
**Ttitle**
Post-Digital Therapy to Repair the Sense of Belonging
